### ISSUE TITLE
"30minutes" is tokenized as "30m inutes"

### ISSUE THREAD

**Issue by rayslxu** [stanford-oval/genie-toolkit#569]

**Comment by gcampax** [comment]
This is literally a bug in the tokenizer, right? We can fix it without significant changes.

**Comment by rayslxu** [comment]
> This is literally a bug in the tokenizer, right? We can fix it without significant changes.

yeah, I will just use this issue to report a list of bugs in tokenization.

**Comment by gcampax** [comment]
Are you going to fix these any time soon?
